app/services/push_notification/mfa_limit_account_locked_event.rb

```diff
@@ -0,0 +1,31 @@
+module PushNotification
+  class MfaLimitAccountLockedEvent
+    EVENT_TYPE =
+      'https://schemas.login.gov/secevent/risc/event-type/mfa-limit-account-locked'.freeze
+
+    attr_reader :user
+
+    def initialize(user:)
+      @user = user
+    end
+
+    def event_type
+      EVENT_TYPE
+    end
+
+    def payload(iss_sub:)
+      {
+        subject: {
+          subject_type: 'iss-sub',
+          iss: Rails.application.routes.url_helpers.root_url,
+          sub: iss_sub,
+        },
+      }
```

Comment:
obvious this is the same in this PR as the others we have, but I think it might be time to find a way to share code between our various event classes, we seem to have two kinds:
1. subject: iss-sub
2. subject: email

at least for the `iss_sub` maybe there's a mixin or something we could include? (I'm not a huge fan of subclassing for simple stuff like this)

```ruby
module IssSubEvent
  def payload(iss_sub:)
    # ...
  end
end

class MfaLimitLockedEvent
  include IssSubEvent
end
```